docs/ehrql-new-tutorial-intro.md

```diff
@@ -81,76 +54,71 @@ Familiarity with:
 would also be helpful,
 but is *not* essential.
 
-#### Software requirements
+## How to work through the Tutorial Series
```

Comment:
suggestion: If we're not using "Tutorial Series" as a proper noun elsewhere, then I don't think we should here.

```suggestion
## How to work through this tutorial series
```